### ISSUE TITLE
Group Permissions

### ISSUE THREAD

**Issue by tophneal** [mtierltd/timetracker#115]
Just wanted to say, thank you for this NC app!

Wanted to note, since it might also be useful for others' use case; it would be great to have permissions/access control for this app based on users/groups.

**Comment by nicoletflr** [comment]
I agree, looks very good. And group acces to a project would be great

**Comment by Gunnar-GO** [comment]
maybe it is missunderstandable, but i requested the same. 

#106
